ruby_ami.gemspec

```diff
@@ -23,9 +23,9 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency %q<celluloid-io>, ["~> 0.13"]
   s.add_runtime_dependency %q<celluloid>, ["~> 0.16.0"]
 
-  s.add_development_dependency %q<bundler>, ["~> 1.0"]
-  s.add_development_dependency %q<rspec>, ["~> 2.5"]
-  s.add_development_dependency %q<cucumber>, [">= 0"]
+  s.add_development_dependency %q<bundler>, [">= 1.0"]
+  s.add_development_dependency %q<rspec>, [">= 3.0"]
+  s.add_development_dependency %q<cucumber>, ["< 3.0"]
```

Comment:
It looks like this was done to make Ruby 1.9.3 happy, but we no longer need to support that ancient version. Should this restriction be lifted?

Reply:
@bklang I was thinking about that too (actually, I completed the cuke upgrade locally and confirmed it's a small effort).

Reason for possibly not doing it.. a handful of cucumber features fail unless we fully embrace cucumber >= 3.0n syntax.  But since the syntax is not backwards compatible, it means that the same features will have to remain different on `support/2.x` vs. `develop`.

I'd lean towards having the two branches have a common denominator, unless it drags down `develop` in some way.

But I can be swayed - I can update this branch to embrace cuke 3.0 if you like it.  What do you think?

Reply:
Your answer makes sense to me. There's no urgency to upgrade cucumber, so let's leave it for now. As you say, we can do this work whenever we have a better reason to do it, and in the meantime we benefit from keeping backports simple. 👍

Reply:
Meant to reply back earlier... ... thanks man.